UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -1373,7 +1373,7 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
 
   updateWritingStyle(writingStyle: WritingStyle) {
     this.writingStyle = writingStyle;
-    this.updateImageSizes()
+    setTimeout(() => this.updateImageSizes());
```

Comment:
I'm assuming you want to do this? this.updateImageSizeTimeout =  setTimeout()?

Reply:
No I don't want to do it here. It doesn't need to get debounced. I needed to debounce the other one as it gets called numerous times once the page loads.